### DISCUSSION TITLE
ResizedImage size is larger than Original Image

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#5405] [Help]
Im using MagickWand methods to resize the image
NewMagickWand
MagickReadImage
MagickResizeImage(Filter : LanczosFilter)
MagickWriteImages
DestroyMagickWand

Original Image Dimension : 2788 X 1382
Original Image Format : jepg
Original Image Size : 3.7MB

Resizing to following dimension : 1600 X 793
Resized Image Format : jepg
Getting resized size as 54MB

Im actually downsizing the image 
Why resized size is greater than the original size ?
What could be possible reason for this large resized image size and how we can handle this better ?

**Comment by snibgo** [comment]
JPEG involves lossy compression. You don't say the format of your output file. Is it JPEG? If so, have you tried different quality settings? Other settings can also be used, for example "`-define jpeg:extent=400KB`". See https://imagemagick.org/script/defines.php .

**Reply by programmerarunkumar** [reply]
Yes output image format is JPEG

**Reply by programmerarunkumar** [reply]
What may be possible reason for resized image size is higher than the original image size

**Comment by snibgo** [comment]
The likely reason is a different "-quality" setting.

**Reply by programmerarunkumar** [reply]
But While resizing the image, I haven't specifed the quality setting 
What is the default quality setting IM will use ?

**Reply by programmerarunkumar** [reply]
> The likely reason is a different "-quality" setting.

Can you elaborate about this 
which means "-quality" setting will be different for original and resized image ?
how "-quality" setting is applied for resized image ?

**Reply by fmw42** [reply]
IM uses the input image quality if it exists otherwise 92.

**Comment by programmerarunkumar** [comment]
When analysing for PNG image format , resized image size is larger than the original image 
What could be possible reason for this ?

**Reply by fmw42** [reply]
please post your command line and your input and output images

PNG is a compressed format. Different compressions can affect the resulting image size. Some png tools can compress better than others. So if you start with a very well compressed png and then process and save with a less well compressed setting, the file size can get larger.

**Reply by programmerarunkumar** [reply]
Using the below command to resize the png image
convert image.png -resize 1000X1000  image1.png

**Comment by snibgo** [comment]
Resizing can also add new colours. Perhaps the input had less then 256 colours that compress well, and resizing adds hundreds more colours that will compress less well.
